tensorboard/webapp/widgets/line_chart_v2/lib/renderer/renderer_types.ts

```diff
@@ -88,6 +98,10 @@ export interface TrianglePaintOption extends PaintOption {
   size: number;
 }
 
+export interface TrapezoidPaintOption extends PaintOption {
+  width: number;
```

Comment:
iiuc, the meaning of this `width` option is the length of the "slanted side" of the trapezoid?

I find it a bit strange to specify this slanted length, rather than the "height" or "altitude" of the trapezoid.  Although both are two ways to express what is functionally the same parameter, I think the altitude, used in the calculations below, is a bit more natural.

Otherwise if specifying the slanted length is preferred, could we rename this variable to `slantedSideLength`?

Reply:
Ah, I did not mean to use "width" as the slant side length. Changed it to `altitude` instead.